dev-qt/qtdeclarative/qtdeclarative-5.15.5.9999.ebuild

```diff
@@ -47,6 +47,8 @@ src_prepare() {
 }
 
 src_configure() {
+	filter-flags "-Os" # bug 840861
```

Comment:
ok, but shouldn't we replace with `-O2` instead of just filtering? I might be totally missing something, but without any optimization flags the default is `-O0` which is not a good idea.